### PR TITLE
Add libxo

### PR DESCRIPTION
**Pull request by CRKatri**
As always, this is untested.

### CODE REVIEW

File: build_info/libxo-tools.control [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Package: libxo-tools
+Version: @DEB_LIBXO_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
+Homepage: https://github.com/Juniper/libxo
+Section: Utilites
+Priority: optional
+Depends: libintl8, perl, libxo (= @DEB_LIBXO_V@)
```
Comment (Diatrus) [inline build_info/libxo-tools.control:8]: I'm stopping here, a classic Cameron moment

**Review by Diatrus — CHANGES_REQUESTED** [review]

File: build_info/libxo-tools.control (hunk 1 above)
Comment (CRKatri) [reply build_info/libxo-tools.control]: True true.

File: libxo.mk [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+ifneq ($(wildcard $(BUILD_WORK)/libxo/.build_complete),)
+libxo:
+	@echo "Using previously built libxo."
+else
+libxo: libxo-setup
+	cd $(BUILD_WORK)/libxo && ./configure -C \
+		$(DEFAULT_CONFIGURE_FLAGS) \
+		--with-gettext=$(BUILD_BASE)/$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+	$(SED) -i'' '/rpl_/d' $(BUILD_WORK)/libxo/libxo/xo_config.h
```
Comment (Diatrus) [inline libxo.mk:22]: Not immediately sure, but this can probably be fixed with a configure arg. ac_cv_has_rpl_malloc or something

Comment (Diatrus) [reply libxo.mk:22]: Also you should be using GNU sed here, why does it have the '' after the -I?

File: build_info/libxo-tools.control (hunk 1 above)
Comment (TheRealKeto) [reply build_info/libxo-tools.control]: Spell it right then.

**Comment by Diatrus** [comment]
@CRKatri if you misspell utilities one more time I'm auto flagging all your PRs

**Comment by CRKatri** [comment]
:eyes:

**Pull request merged**